examples-extra/basic/sdk/src/generatedNoCheck/Ontology.ts

```diff
@@ -27,7 +28,14 @@ const _Ontology = {
     SimpleInterface: Interfaces.SimpleInterface,
   },
 } satisfies OntologyDefinition<
-  'WeatherStation' | 'BoundariesUsState' | 'Todo' | 'Person' | 'Employee' | 'Venture' | 'ObjectTypeWithAllPropertyTypes'
+  | 'WeatherStation'
+  | 'BoundariesUsState'
+  | 'BuilderDeploymentState'
+  | 'Todo'
+  | 'Person'
+  | 'Employee'
+  | 'Venture'
+  | 'ObjectTypeWithAllPropertyTypes'
```

Comment:
This is probably a semantic merge conflict with https://github.com/palantir/osdk-ts/pull/124, I'll resolve later.

Reply:
Actually I'm going to merge mine, can you eventually rebase and just run codegen before you merge?

Reply:
Yea I can rebase, go for it